crates/sui/src/validator_commands.rs

```diff
@@ -85,18 +85,30 @@ pub enum SuiValidatorCommand {
         file: PathBuf,
         #[clap(name = "gas-budget", long)]
         gas_budget: Option<u64>,
+        /// If true, only print the unsigned transaction and do not execute it.
+        /// This is useful for offline signing.
```

Comment:
What do you think about keeping it the same as in `sui client side`


```suggestion
        /// Instead of executing the transaction, serialize the bcs bytes of the unsigned transaction data
        /// (TransactionData) using base64 encoding, and print out the string <TX_BYTES>. The string can
        /// be used to execute transaction with `sui client execute-signed-tx --tx-bytes <TX_BYTES>`.
```